public/app/types/explore.ts

```diff
@@ -108,7 +104,8 @@ export interface ExploreItemState {
   /**
    * Log query result to be displayed in the logs result viewer.
    */
-  logsResult: LogsModel | null;
+  logsResult?: LogsModel;
```

Comment:
LogsResult is needed as the schema is different but it may be possible to move that inside the logs panel in the future.